src/ripple/rpc/impl/TransactionSign.cpp

```diff
@@ -679,14 +679,11 @@ Json::Value checkFee (
         feeTrack.scaleFeeLoad (feeDefault,
             ledger->fees().base, ledger->fees().units, isUnlimited (role));
     std::uint64_t fee = loadFee;
-    if (ledger->rules().enabled(featureFeeEscalation,
-        config.features))
```

Comment:
I'm assuming you're removing the `featureFeeEscalation` check because fee escalation is already enabled on the network.  If that's the case, I'm wondering if it might be better to clean those out all at once in a separate pull request?  There are a number of similar checks, for example, in TxQ.cpp.  Just a thought.  Your call.